src/open_inwoner/accounts/views/contacts.py

```diff
@@ -31,69 +30,30 @@ def crumbs(self):
 
     def get_queryset(self):
         base_qs = super().get_queryset()
-        base_qs = base_qs.get_extended_contacts_for_user(me=self.request.user)
+
+        base_qs = self.request.user.get_active_contacts()
         type_filter = self.request.GET.get("type")
+
         if type_filter:
-            if type_filter == ContactTypeChoices.contact:
-                base_qs = base_qs.filter(
-                    Q(contact_user__contact_type=type_filter)
-                    | Q(contact_user__isnull=True)
-                )
-            else:
-                base_qs = base_qs.filter(
-                    Q(contact_user__contact_type=type_filter)
-                    | Q(created_by__contact_type=type_filter)
-                )
-        return base_qs
+            base_qs = base_qs.filter(contact_type=type_filter)
+
+        return base_qs.order_by("-date_joined")
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+
+        user = self.request.user
+        context["pending_approvals"] = user.get_contacts_for_approval()
```

Comment:
I did not apply a filter because I thought since it's not a contact yet we shouldn't give extra information (except the email). I will change this if we should.

Reply:
Discussed during weekly meeting, we will not apply a filter here.